longevity_large_partition_test.py

```diff
@@ -12,7 +12,9 @@ def test_large_partition_longevity(self):
 
     def pre_create_large_partitions_schema(self, compaction_strategy=CompactionStrategy.SIZE_TIERED.value):
         node = self.db_cluster.nodes[0]
-        create_table_query = create_scylla_bench_table_query(compaction_strategy=compaction_strategy)
+        table_setup_seed = self.params.get('nemesis_seed')
```

Comment:
@roydahan asked to not use the nemesis seed, but create a seed for this use case

Reply:
It's also shouldn't be set, so it will continue being random unless we want to set it for reproduce.